### ISSUE TITLE
Unable to build qumu

### ISSUE THREAD

**Issue by TheDarkerItGetsTheMoreHiddenItIs** [devos50/qemu#100]
PS C:\Users\LENOVO\build> ../configure  --enable-sdl --disable-cocoa --target-list=arm-softmmu --disable-capstone --disable-slirp --disable-pie --extra-cflags=-I/mingw64/include/openssl --extra-ldflags='-L/mingw64/lib -lcrypto' --disable-stack-protector
../configure: "The term '../configure' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again."

**Comment by Dragan232** [comment]
On Windows you are to use [MSYS2 MINGW64](https://www.msys2.org/).

**Comment by MrDiagnose** [comment]
u can't run configure script inside powershell install https://www.msys2.org/ and run it there

**Comment by devos50** [comment]
Indeed, you need a different toolchain on Windows. Also, please note that the `build` directory has to be made in the root of the source code. Based on your path (`C:\Users\LENOVO\build`), I'm not sure if that's the case.
